src/main/java/com/termos/scooterrental/SpringBootConsoleApplication.java

```diff
@@ -0,0 +1,196 @@
+package com.termos.scooterrental;
+
+import com.termos.scooterrental.model.Rent;
+import com.termos.scooterrental.model.Scooter;
+import com.termos.scooterrental.repository.RentRepository;
+import com.termos.scooterrental.time.TimeUtils;
+import com.termos.scooterrental.model.ScooterDock;
+import com.termos.scooterrental.model.UserAccount;
+import com.termos.scooterrental.repository.ScooterDockRepository;
+import com.termos.scooterrental.repository.ScooterRepository;
+import com.termos.scooterrental.repository.UserAccountRepository;
+import lombok.AllArgsConstructor;
+import lombok.Lombok;
+import org.springframework.boot.Banner;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+import static java.lang.System.setOut;
+
+@SpringBootApplication
+@AllArgsConstructor
+public class SpringBootConsoleApplication implements CommandLineRunner {
+    ScooterDockRepository scooterDockRepository;
+    ScooterRepository scooterRepository;
+    UserAccountRepository userAccountRepository;
+    RentRepository rentRepository;
+    static Scanner scan = new Scanner(System.in);
+
+    public static void main(String[] args) throws Exception {
+
+        //disabled banner, don't want to see the spring logo
+        SpringApplication app = new SpringApplication(SpringBootConsoleApplication.class);
+        app.setBannerMode(Banner.Mode.OFF);
+        app.run(args);
+
+        //SpringApplication.run(SpringBootConsoleApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+
+        Scanner input = new Scanner(System.in);
+        int inputInt;
+
+        while (true) {
+            System.out.println(" ");
```

Comment:
Zamiast drukować całe menu linijka po linijce możesz wydzielić to do oddzielnej metody (będzie to wtedy znacznie czytelniejsze).

Również nie ma sensu wywoływać metody `System.out.println` wielokrotnie jeśli możesz skorzystać z konkatenacji String